src/Health/Tests/Endpoints.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Tribe\Extensions\Promoter\Health\Tests;
+
+use Tribe\Extensions\Promoter\Health\Critical_Exception;
+use Tribe__Promoter__Connector;
+
+class Endpoints implements Runnable {
+
+	public function run() {
+		$this->connector = tribe( 'promoter.connector' );
```

Comment:
Add `@var`

Reply:
This one has been already defined at the class level of the variable instead so the type and such would be available.